src/migrations/20181205133512_added_todos.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @param  {object} knex
+ * @return {Promise}
+ */
+export function up(knex) {
+  return knex.schema.createTable('todos', table => {
+    table.increments();
+    table
+      .timestamp('created_at')
+      .notNull()
+      .defaultTo(knex.raw('now()'));
+    table.timestamp('updated_at').notNull();
+    table.string('title').notNull();
+    table.string('status').notNull();
+    table.string('username').notNull().unique();
```

Comment:
The `unique` constraint doesn't make sense here.
This would mean a user can create only one todo note. You should remove this constraint.

Reply:
aaa dai yo paila ko commit vayecha aaile UUID haleko chu. Yo TODOS ko sab USER bata copy paste haneko parcha xD